### PR TITLE
PRS13889 90

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Scribe and  commissioner of Abb. 64.

### CODE REVIEW

File: new/PRS13890Sankoras.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <revisionDesc>
+            <change who="DN" when="2022-09-05">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሰንኮሪስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Sankoris</persName>
```
Comment (eu-genia) [inline new/PRS13890Sankoras.xml:49]: Sankoris or Sankorās as was proposed in https://github.com/BetaMasaheft/Documentation/issues/2226?

File: new/PRS13890Sankoras.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="other"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1500" notAfter="1599"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:isCopierOf" active="PRS13890Sankoras" passive="BNFabb64"/>
```
Comment (eu-genia) [inline new/PRS13890Sankoras.xml:58]: relation correct?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13890Sankoras.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13890Sankoras.xml:49]: Sankoris.

File: new/PRS13890Sankoras.xml (hunk 2 above)
Comment (eu-genia) [reply new/PRS13890Sankoras.xml:58]: in the issue he was listed as commissioner

**Review by eu-genia — APPROVED** [review]

**Pull request merged**